tests/bats/govcms-vet.bats

```diff
@@ -23,7 +23,7 @@ setup() {
 }
 
 vet() {
-  "$CUR_DIR"/scripts/govcms-vet
+  "$CUR_DIR"/scripts/govcms-vet "$(yq read .version.yml type)" develop
```

Comment:
This is to ensure we are testing latest `develop` upstream. The version found in `.version.yml` (e.g `2.0.0`) cannot be tested without a tagged release otherwise.

Reply:
Yeah that's good. This bats test is just testing the sanity of the script, and should only do it against a known good state (ie `develop` branch).